ssg/pep8.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3-pep8 *.py
```

Comment:
I don't think that a standalone script that runs pep8 should be added here. Seems like it should be added to CMake.

Reply:
yeah, IMO this should be a ctest check

Reply:
Yeah, sorry, this was a personal artifact that leaked in when I bulk added `ssg`. I want to add this as a `ctest` eventually but haven't gotten around to it yet.